Makefile

```diff
@@ -32,5 +32,10 @@ docker-build:
 	docker run --network=host -v $(shell pwd):/source -v $(GOPATH)/pkg/mod:/go/pkg/mod golang:1.13-alpine /bin/sh \
 	-c "cd /source && apk add git gcc musl-dev make && make build"
 
+endtoend-test:
+	cd e2e && \
+	docker build -t c.txtdirect.org/tester:0.0.1 . && \
```

Comment:
What about using `c.txtdirect.org/tester:0.0.1-dirty` to make clear this is a build from code, that's not meant to be used anywhere?
Also do we want to version this and if yes how?

Reply:
I don't think versioning it would be necessary. Should we use something like `c.txtdirect.org/tester:dirty` or `c.txtdirect.org/tester` is enough?

Reply:
`c.txtdirect.org/tester:dirty` sounds good.